tailwind.config.js

```diff
@@ -7,6 +7,19 @@ export default {
   theme: {
     extend: {},
   },
-  plugins: [],
+  daisyui: {
+    themes: [
+      {
+        mytheme: {
```

Comment:
This is where we set a custom DaisyUI theme using the style guide created by our designers. One of the benfits of DaisyUI is that its built-in color names (primary, secondary, accent) match design best practice for naming conventions (as you can see from Kayla's Figma file). To see what each of these colors mean - refer to the DaisyUI color docs linked in the PR.